viewing_party/party.py

```diff
@@ -1,23 +1,188 @@
-# ------------- WAVE 1 --------------------
 
+# ------------- WAVE 1 --------------------
+# Creates a dictionary that holds movies with title, genre, and rating
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title,
+                 "genre": genre,
+                 "rating": rating}
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        return new_movie

```

Comment:
I'd suggest flipping the order of the conditions around to handle the special case (ie, checking if something is falsy) first, so that the code can quickly check and get out of there. This arrangement is referred to as a "guard clause" or a "guard condition." Notice that it lets us move the main focus of the function (creating a dictionary from the parameters) out of an indented block, letting it stand out more clearly.

Another thing is, if we flip the conditionals, the program doesn't waste memory space making a dictionary that won't be returned if the parameters turn out to be falsy

```suggestion
    if title == None or genre == None or rating == None:
        return None
    new_movie = {
                 "title": title,
                 "genre": genre,
                 "rating": rating
                 }

    return new_movie

```